### ISSUE TITLE
[xray] Chart defaults do not meet system requirements

### ISSUE THREAD

**Issue by TJM** [jfrog/charts#1178]
**Is this a request for help?**: No

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

**Version of Helm and Kubernetes**:

```
[tmcneely@local dns] (⎈ |den3sbx:artifactory)$ helm version
version.BuildInfo{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"}
[tmcneely@local dns] (⎈ |den3sbx:artifactory)$ kubectl version
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.1", GitCommit:"206bcadf021e76c27513500ca24182692aabd17e", GitTreeState:"clean", BuildDate:"2020-09-09T19:10:21Z", GoVersion:"go1.15.1", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.12", GitCommit:"e2a822d9f3c2fdb5c9bfbe64313cf9f657f0a725", GitTreeState:"clean", BuildDate:"2020-05-06T05:09:48Z", GoVersion:"go1.12.17", Compiler:"gc", Platform:"linux/amd64"}
```

**Which chart**:
xray

**What happened**:
PostgreSQL ran out of disk space (50GB)

**What you expected to happen**:
The chart should provision the application with enough disk space to work.

**How to reproduce it** (as minimally and precisely as possible):

* Deploy Artifactory and Xray, enable sync

**Anything else we need to know**:

https://www.jfrog.com/confluence/display/JFROG/System+Requirements

* Xray should have 100GB PVC
* XrayDB should have 300GB  PVC

**Comment by TJM** [comment]
NOTE: in values-small.yaml, I see an indication of persistence for xray server itself, but its at the wrong indentation level to be effective:
https://github.com/jfrog/charts/blob/010abb54406ff64feb25ea7e0141512ae55f6941/stable/xray/values-small.yaml#L59-L60

**Comment by chukka** [comment]
@TJM  we were aware of this. This is already addressed in existing  PR #1099 (Since this is breaking change for existing customers , we wanted to do it with major version bump)

**Comment by chukka** [comment]
> NOTE: in values-small.yaml, I see an indication of persistence for xray server itself, but its at the hash level to be effective (obviously values-medium.yaml and values-large.yaml have the same indentation issue):
> https://github.com/jfrog/charts/blob/010abb54406ff64feb25ea7e0141512ae55f6941/stable/xray/values-small.yaml#L59-L60
> 
> Should be:
> 
> ```yaml
> common:
>   persistence:
>     size: 100Gi
> ```
> 
> ... but that should actually be the default values.yaml size.

Fixed in #1099

**Comment by TJM** [comment]
thanks!
